src/index.js

```diff
@@ -3,31 +3,13 @@ import styled from 'styled-components'
 import {
   space,
   color,
-  width,
-  height,
-  flex,
-  order,
-  alignSelf,
-  flexWrap,
-  flexDirection,
-  alignItems,
-  justifyContent,
-  fontSize,
-  fontFamily,
-  fontWeight,
-  textAlign,
-  lineHeight,
-  letterSpacing,
-  borders,
-  borderColor,
-  borderRadius,
+  layout,
+  flexbox,
+  typography,
+  border,
   buttonStyle,
-  boxShadow,
-  backgroundImage,
-  backgroundSize,
-  backgroundPosition,
-  backgroundRepeat,
-  opacity,
+  shadow,
+  background,
   variant,
 } from 'styled-system'
```

Comment:
I think it would be cool if rebass exported all the styled system fns (like `apollo-boost` does with some of the apollo internals, e.g.) - that way, once you've installed styled-components and rebass you don't have to worry about conflicting internal dependencies.

What do you think @jxnblk? Is this too much to ask?

Reply:
Great idea! Added